dockerized

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -euxo pipefail
+
+# We always want the latest image
+docker pull ubuntu:20.04
+
+# CACHEBUST means this will apply the updates once a day
+docker build . -t distributions --build-arg CACHEBUST=`date --iso-8601=date` 
```

Comment:
```suggestion
docker build . -t distributions --build-arg CACHEBUST=`date --iso-8601=date` --build-arg USER_UID=$(id -u "$USER")
```
The docker container will not have permission to write to the mapped volumes since it is running as the container's `asdf` user, which has the uig:gid 1000:1000.  This means that the user needs to make their `distributions` directory (mapped into the container as `/build`) accessible to that uid. **Or, better, create the `asdf` user with the same uid that the user invoking this script has.** This passes the user's uid into the Dockerfile as a build arg.